docs/source/api/v2/servers_id_deliveryservices.rst

```diff
@@ -239,3 +239,81 @@ Response Structure
 
 
 .. [#tenancy] Only the :term:`Delivery Services` visible to the requesting user's :term:`Tenant` will appear, regardless of their :term:`Role` or the :term:`Delivery Services`' actual 'server assignment' status.
+
+``POST``
+========
+Assign an arbitrary number of delivery services to a single server.
+
+:Auth. Required: Yes
+:Roles Required: "admin" or "operations"
+:Response Type:  Object
+
+Request Structure
+-----------------
+.. table:: Request Path Parameters
+
+	+------+----------+---------------------------------------------------------------------------------------------+
+	| Name | Required | Description                                                                                 |
+	+------+----------+---------------------------------------------------------------------------------------------+
+	| id   | Yes      | The integral, unique identifier of the server that you want to assign delivery services to. |
+	+------+----------+---------------------------------------------------------------------------------------------+
+
+.. table:: Request Query Parameters
+
+	+---------+----------+------------------------------------------------------------------------------------------+
+	| Name    | Required | Description                                                                              |
+	+=========+==========+==========================================================================================+
+	| replace | Yes      | Must be a 1, true, 0, or false. Although this query parameter is required, it is unused. |
+	+---------+----------+------------------------------------------------------------------------------------------+
+
+The request body is an array of IDs of delivery services that you want to assign to the server. The array can be empty, but it must be provided.
```

Comment:
doesn't need to be linked again but should be "Delivery Services" not "delivery services"

Reply:
Changed to Delivery Services in cfbcfe347a.